### DISCUSSION TITLE
Docker support

### DISCUSSION THREAD

**Discussion by SpudmanWP** [Umio-Yasuno/amdgpu_top#106] [Ideas]
I have a TrueNAS that I have Handbrake on in order to transpose videos.  However, I have no way know knowing if it's using the CPU or GPU cores of the APU (a Ryzen PRO 5650G).  Given that the 5650G GPU cores support hardware Encode/Decode in h265, I need to make sure the GPU side of the APU is doing the work.

To that end, I need a proper CPU/GPU/APU analysis tool to see what is going on.  I am hoping that your tool running in a Docker container would work but I do not know enough about linux to build my own.

What are the chances of you putting out a NAS-compatible Docker version with a Docker Compose file?

**Comment by Umio-Yasuno** [comment]
> What are the chances of you putting out a NAS-compatible Docker version with a Docker Compose file?

No.  
I don't understand why I need to publish and maintain Docker files.
